docs/embedding/sdk/questions.md

```diff
@@ -139,26 +139,169 @@ To customize the layout, use namespaced components within the `InteractiveQuesti
-| `BackButton`               | The back button, which provides `back` functionality for the InteractiveDashboard                                            |
-| `Filter`                   | A set of badges for adding, viewing, and editing the filters of the current question                                         |
-| `FilterPicker`             | Picker for adding a new filter to the question                                                                               |
-| `FilterDropdown`           | A button which has a dropdown for adding, viewing, and editing the filters of the current question                           |
-| `ResetButton`              | The button used to reset the question after the question has been modified with filters/aggregations/etc                     |
-| `Title`                    | The question's title                                                                                                         |
-| `SaveButton`               | Button for saving the question.                                                                                              |
-| `Breakout`                 | A set of badges for adding, viewing, and editing the groupings of the current question                                       |
-| `BreakoutDropdown`         | A button which has a dropdown for adding, viewing, and editing the groupings of the current question                         |
-| `Summarize`                | A set of badges for adding, viewing, and editing the groupings of the current question                                       |
-| `SummarizeDropdown`        | A button which has a dropdown for adding, viewing, and editing the groupings of the current question                         |
-| `Notebook`                 | The Notebook editor that allows for more filter, aggregation, and custom steps                                               |
-| `NotebookButton`           | The button used in the default layout to open the Notebook editor. You can replace this button with your own implementation. |
-| `QuestionVisualization`    | The chart visualization for the question                                                                                     |
-| `QuestionSettings`         | The settings for the current visualization                                                                                   |
-| `QuestionSettingsDropdown` | A settings button that contains the `QuestionSettings` component in the dropdown                                             |
-| `ChartTypeDropdown`        | A dropdown containing the possible visualization options for the current question                                            |
+#### `InteractiveQuestion.BackButton`
+> A navigation button that returns to the previous view. Only renders when `onNavigateBack` is available in context.
```

Comment:
I think the use of quotes makes the docs feel a bit odd. I search the SDK doc and we never use quotes anywhere.
![image](https://github.com/user-attachments/assets/b99d50fb-2085-4918-b4a6-aab6e07e1574)

I think we should just simply unquote everything.